README.md

```diff
@@ -18,13 +18,25 @@ pnpm install
 ### Build
 
 ```shell
-forge build
+pnpm build
+# or 
+# forge build
 ```
 
 ### Test
 
 ```shell
-forge test
+pnpm test 
+# or 
+# forge test --match-path "test/**/*.sol" or 
+```
+
+### Test for scripts
+
+```shell
+pnpm test:script
```

Comment:
Could you add `--threads 1` to the test command because the tests for scripts might fail with multiple threads due to conflicts among env settings.

Reply:
@SoraSuegami pnpm test:script command already has this option.

Reply:
@SoraSuegami I've added thread 1 option to foundry.toml